### PR TITLE
dxScheduler - fix throw exception on onAppointmentFormOpening(T812654)

### PR DESCRIPTION
**Pull request by andrewmakarov**

### CODE REVIEW

**Review by jsdmitry — DISMISSED** [review]

File: js/ui/scheduler/ui.scheduler.appointment_form.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ const SchedulerAppointmentForm = {
                     type: "required"
                 }],
                 editorOptions: {
-                    type: allDay ? "date" : "datetime",
+                    // type: allDay ? "date" : "datetime",
```
Comment (Olexi) [inline js/ui/scheduler/ui.scheduler.appointment_form.js]: Can you remove this comment?

File: js/ui/scheduler/ui.scheduler.appointment_form.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ const SchedulerAppointmentForm = {
...
```
Comment (Olexi) [inline js/ui/scheduler/ui.scheduler.appointment_form.js]: Can you remove this comment?

File: testing/functional/tests/scheduler/appointmentForm/T812654.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { createWidget } from '../../../helpers/testHelper';
```
Comment (Olexi) [inline testing/functional/tests/scheduler/appointmentForm/T812654.ts:1]: Why the name of file is based on the ticket, but not on the point of it?

File: testing/functional/tests/scheduler/appointmentForm/T812654.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        {
+            show1: false,
+            text: "Website Re-Design Plan",
+            startDate: new Date(2017, 4, 22, 9, 30),
+            endDate: new Date(2017, 4, 22, 11, 30)
+        }
+    ]
+}, true);
+
+fixture `Scheduler: T812654`
```
Comment (Olexi) [inline testing/functional/tests/scheduler/appointmentForm/T812654.ts:40]: Why the fixture named after ticket ID?

File: testing/tests/DevExpress.ui.widgets.scheduler/appointmentPopup.tests.js [hunk 5]
```diff
@@ -13,8 +13,77 @@ import resizeCallbacks from "core/utils/resize_callbacks";
...
+            height: 600,
+            width: 600
+        });
+    };
+
+    QUnit.test("Appointment popup should work properly", assert => {
+        const NEW_EXPECTED_SUBJECT = "NEW SUBJECT";
+        const scheduler = createScheduler();
+
+        assert.notOk(scheduler.appointmentPopup.isVisible(), "appointment popup should be invisible in on init");
```
Comment (Olexi) [inline testing/tests/DevExpress.ui.widgets.scheduler/appointmentPopup.tests.js]: Please name assert description with Capital letter

**Review by jsdmitry — APPROVED** [review]

**Review by Olexi — APPROVED** [review]

**Pull request merged**